src/fmeval/transforms/summarization_accuracy_metrics.py

```diff
@@ -126,26 +146,31 @@ class MeteorScore(SummarizationAccuracyMetric):
 
     def __init__(
         self,
-        target_output_keys: List[str],
-        model_output_keys: List[str],
         output_keys: List[str],
-        allow_duplicate_input_keys,
+        model_output_keys: List[str],
+        allow_duplicate_input_keys: bool,
```

Comment:
What's the default for `allow_duplicate_input_keys`? Will this change break existing code?

Reply:
It doesn't have a default value. There's no change right now other than adding a type annotation. As an aside, with the new changes, we no longer need this argument. But to keep things backward-compatible, we should leave it for now.